runtime/doc/various.txt

```diff
@@ -277,6 +277,7 @@ g8			Print the hex values of the bytes used in the
 <			The screen is not redrawn then, thus you have to use
 			CTRL-L or ":redraw!" if the command did display
 			something.
+			Also see |:terminal|.
```

Comment:
This makes more sense on line 236. I suggest this edit:

      :!{cmd}			Execute {cmd} with 'shell'. See also |:terminal|.